lib/api/user.ts

```diff
@@ -49,6 +49,7 @@ export async function getUser(username: string): Promise<UserProps | null> {
     { projection: { _id: 0, emailVerified: 0 } }
   );
   if (results) {
+    // @ts-ignore
```

Comment:
What makes the `@ts-ignore` necessary here? Is there a way it can be avoided?

Reply:
fixed in https://github.com/vercel/mongodb-starter/pull/26/commits/70cce80061ab3ba632e8a873a70b8e824bcfdc70 ! 👍